coralproject/talk
# Smooth Scroll on Moderation

## What does this PR do?
- Adds smooth Scrolling

## How do I test this PR?
- Go to moderation
- Press J or K and move
- It should be smooth
